### DISCUSSION TITLE
Galaxy Brain Achievement Test

### DISCUSSION THREAD

**Discussion by keyiflerolsun** [Schweinepriester/github-profile-achievements#45] [Show and tell]
Should I start two discussions or answer two discussions ?:)

**Comment by Schweinepriester** [comment]
become the selected answer to two discussions

**Reply by nel0x** [reply]
Getting the Galaxy Brain badge doesn't seem to be possible anymore.
You only get "x discussions answered" now in your "highlights" ...

**Reply by fabiomfrade** [reply]
This badge was not replace by any other?

**Reply by ojeffersondev** [reply]
Ainda está funcionando?

**Reply by fabiomfrade** [reply]
aparentemente não está mais não, irmão

**Reply by dominikvytisk** [reply]
❤️

**Reply by bagaaz** [reply]
How can i get most fastad the answers ?

**Reply by vivianrcc** [reply]
mas entao o selo aparece em qualquer das duas opçoe?s

**Reply by Arturs1123** [reply]
Is it really?

**Reply by dougcostadev** [reply]
acho que não da mais certo

**Reply by Arturs1123** [reply]
I think it still works

**Reply by HDTN** [reply]
🙌

**Reply by minatisleeping** [reply]
nice :333

**Comment by ItzJustNick** [comment]
I think you need to answer 2 disscusions and then become the accepted answer on both to get the achivement. Hope this helps.

**Reply by serAnkii** [reply]
You have to answer the discussion questions.

**Reply by Noctino52** [reply]
not work

**Reply by TechBond7** [reply]
not working

**Reply by vivianrcc** [reply]
acredito que so sirva se for em respostas de repositorios com alguem aceitando suas respostas

**Reply by gabopedrosa** [reply]
Hey!

**Reply by Arturs1123** [reply]
I need only 2 discussion?

**Reply by imtushaarr** [reply]
not work?

**Reply by Arturs1123** [reply]
It is working now.

**Reply by minatisleeping** [reply]
nice :333

**Comment by nel0x** [comment]
As said, don't know exactly why but that badge is **not** available anymore. @Schweinepriester

**Reply by Schweinepriester** [reply]
are you able to link your discussions?

**Reply by nel0x** [reply]
yes, see e.g.
https://github.com/nel0x/zsh-config/discussions/10
https://github.com/nel0x/zsh-config/discussions/7

**Reply by Schweinepriester** [reply]
Mhhh so I would think that https://github.com/nel0x/zsh-config/discussions/7 counts, because it's not started by yourself and OP selected you as the answer (as opposed to https://github.com/nel0x/zsh-config/discussions/9 where you selected yourself, not sure if that counts).

If I'm not mistaken there was also talk about that there has to be more than one comment in the discussion before you are selected as the answer for it to count (potentially by another user?) - let me check my two that counted.

EDIT while initially typing, I saw you edited 9 to https://github.com/nel0x/zsh-config/discussions/10, mhhh maybe it's true about the two comments 🤔

If we don't solve it now, I'll keep an eye out for anybody with Galaxy Brain awarded after now.

I mean could also be that they broke it or there are some hidden timing functions, like the discussion has to be live for x minutes before it's able to count or something…

**Reply by Schweinepriester** [reply]
Okay, digging up https://github.com/Schweinepriester/github-profile-achievements/discussions/9#discussioncomment-2927413:
- https://github.com/Schweinepriester/github-profile-achievements/discussions/1 didn't have another comment when it was selected as the answer; I commented 11 min after the discussion was started
- https://github.com/Schweinepriester/github-profile-achievements/discussions/9#discussioncomment-2927352 also didn't have another comment when it was selected; I commented 4 min after the discussion was started
- We know nothing about the precise time they were selected as the answer

Mhhh

**Reply by nel0x** [reply]
Yeah, interesting, I'll try a more realistic simulation and submit an answer in a few hours.

**Reply by nel0x** [reply]
@Schweinepriester 
https://github.com/Schweinepriester/github-profile-achievements/discussions/59
https://github.com/Schweinepriester/github-profile-achievements/discussions/60

If these two discussions are not the most legit ones ....
Don't know what happened with that badge but it's **not ernable** at the moment (since a few days).

![image](https://user-images.githubusercontent.com/87320197/175777531-4f60a06f-746c-4ca0-96e7-33b8d520b8fe.png)

But you can see it in my highlights (was on "8 discussions answered) before ...

**Reply by Schweinepriester** [reply]
I take it you were temporarily selected as in the answer in #59 as well?

VERY curious that, for me, you are back to 9 discussions answered:
![image](https://user-images.githubusercontent.com/2644614/175788366-81616739-3a5d-49b9-9ac3-ac200c500a0b.png)

because when we were testing in #9 (deselecting my comment) my count didn't went down (back then).

I bet they are tinkering with it currently!
Maybe they have broken it or there are criteria now that we don't know… it would also be kinda funny and honoring if you couldn't do it anymore on this repo :D Maybe, like, only active repos, like organically?

I'll try to search around a bit and if I don't find anything I'll try to ask them directly if they know it still works (I'll post the link then).

**Reply by Schweinepriester** [reply]
oookay, we already have counterproof:
- https://github.com/github-community/community/discussions/19433
- https://github.com/github-community/community/discussions/19434
- https://github.com/RobertGoodman08
- https://github.com/RobertGoodman08?achievement=galaxy-brain&tab=achievements

They just unlocked Galaxy Brain ~2 h ago :o 

I'll including a bunch of screenshots below to document the current state, but it's probably going to be a bit unwieldy.

Maybe hang around https://github.com/github-community/community/discussions/categories/profile and answer questions there?

Also what I said before isn't correct:

> We know nothing about the precise time they were selected as the answer

I noticed the "Events" section bottom right (on desktop) in each discussion - there we have a precise time.

---

![image](https://user-images.githubusercontent.com/2644614/175788795-8ee36ae5-c097-4374-9f5d-8d265c5e724f.png)

The texts translated using https://www.deepl.com/

> Как получить значок Galaxy Brain?

=>

> How do I get a Galaxy Brain badge?

plus

> Ответьте как минимум на 2 обсуждения.
Обратите внимание, вы НЕ МОЖЕТЕ отмечать свои собственные ответы на свои собственные вопросы. Они должны быть отмечены другим пользователем.
Обсуждения должны быть в общедоступном хранилище.
В настоящее время некоторые пользователи наблюдают ошибку, из-за которой пометка ваших собственных ответов может помешать вам получить значок вместо этого.
Надеюсь, этот ответ помог вам, и всего наилучшего в получении достижения! :)

=>

> Answer at least 2 discussions.
Please note, you CANNOT mark your own answers to your own questions. They must be marked by another user.
Discussions must be in the public repository.
Currently, some users are seeing a bug where marking your own answers may prevent you from getting a badge instead.
I hope this answer helped you, and all the best in getting the achievement! :)

![image](https://user-images.githubusercontent.com/2644614/175788852-93474e15-f25d-4a2d-93fe-d9b8e70c5bc6.png)

![image](https://user-images.githubusercontent.com/2644614/175788915-df039f87-a202-4b80-a76c-3d8b23c56836.png)

**Reply by nel0x** [reply]
> VERY curious that, for me, you are back to 9 discussions answered

eeexactly!

> Currently, some users are seeing a bug where marking your own answers may prevent you from getting a badge instead.
I hope this answer helped you, and all the best in getting the achievement! :)

Could be that it has something to do with that ... whatsoever

> it would also be kinda funny and honoring if you couldn't do it anymore on this repo :D

Yes, of course! Thanks for all your inversted energy @Schweinepriester, I'll keep trying ;D

**Reply by nel0x** [reply]
Update: 
> Currently, some users are seeing a bug where marking your own answers may prevent you from getting a badge instead.
I hope this answer helped you, and all the best in getting the achievement! :)

That was the bug! see:
https://github.com/Schweinepriester/github-profile-achievements/discussions/58

Solved it now :D

**Reply by Arturs1123** [reply]
How can i get git badges?

[a comment by an automated account is not shown]

**Comment by GempitaRizki** [comment]


**Comment by adiii21** [comment]
How can i get that?

**Reply by ChetVladimira** [reply]
same

**Reply by the1Riddle** [reply]
@adiii21 you need to comment two answers that are marked as answers

**Reply by minatisleeping** [reply]
> @adiii21 you need to comment two answers that are marked as answers

ýe

**Comment by HarshDekate** [comment]
To earn the "Galaxy brain" badge, you should aim to answer two discussions by providing helpful and insightful responses that the community finds valuable. This typically involves engaging in existing discussions and contributing to them in a meaningful way.

**Reply by imtushaarr** [reply]
yes

**Reply by minatisleeping** [reply]
> To earn the "Galaxy brain" badge, you should aim to answer two discussions by providing helpful and insightful responses that the community finds valuable. This typically involves engaging in existing discussions and contributing to them in a meaningful way.
ýe

**Comment by HarshDekate** [comment]
give  star to my repos i will give you badges

**Reply by the1Riddle** [reply]
let me just create a discussion for this

**Reply by the1Riddle** [reply]
be sure to check it out

**Comment by Nirzak** [comment]
I think you need to get your answer accepted by the person who asked the question. Then you will get the badge.

**Comment by rizwan3d** [comment]
Is that a real thing.

**Comment by zahid60** [comment]
how can I get this?

**Comment by KaibutsuSama** [comment]
❤️

**Comment by Noctino52** [comment]
<3

**Comment by hadamalush** [comment]
Hello everyone!

**Comment by TechBond7** [comment]
Hello!

**Comment by Churchillk** [comment]
🤣🤣😍

**Comment by gabopedrosa** [comment]
Heyt sure

**Comment by gabopedrosa** [comment]
Accet

**Comment by Elmer-Solis** [comment]
hello

**Comment by Arturs1123** [comment]
How can I get github badges?

**Comment by chenzejun** [comment]
hello

**Reply by Arturs1123** [reply]
Hi

**Reply by imtushaarr** [reply]
hi

**Comment by glucozo-100** [comment]
❤️

**Comment by imtushaarr** [comment]
answer

**Comment by Nirzak** [comment]
So the main answer is you need two accepted answers in a discussion. then you will get galaxy brain achievement.

**Comment by rahpt** [comment]
test

**Reply by rahpt** [reply]
ok

**Comment by Saqib198** [comment]
To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by Eris-js** [comment]
Tôi cũng nghĩ tôi cần được 2 câu trả lời
